src/views/User/User.tsx

```diff
@@ -29,13 +29,11 @@ import {PlayerIcon} from 'PlayerIcon';
 import {GameList} from "GameList";
 import {Player} from "Player";
 import * as preferences from "preferences";
-import {updateDup, alertModerator, getGameResultText, ignore} from "misc";
```

Comment:
don't think you meant to remove the alertModerator import here? it wasn't related to displaying the endstate of the game

Reply:
You are correct that it is unrelated, but this it is an `alertModerator` is an unused import (courtesy of my linter).  I'm guessing it fell off whenever that functionality was moved into PlayerDetails.

If you think I should remove that kind of stuff for better chance at being approved, I will, but I thought it was a good idea to tidy up while I was looking at that file.

Reply:
just caught my eye, I can't recommend one way or another... anoek may advise whether cleanup should have its own PR or not, frankly it's not my call I just wanted to see why it was included

Reply:
If you're curious, [commit b88bc3369f17e5036d309ff96cf1e559b83bc2e1](https://github.com/online-go/online-go.com/commit/b88bc3369f17e5036d309ff96cf1e559b83bc2e1#diff-3634b4c362041964a17503f2ba9d107297f02376af3788b4648644d880de5887) is when `alertModerator` was removed from the profile page :)